client/my-sites/checkout/src/components/wp-checkout-order-summary.tsx

```diff
@@ -122,6 +126,48 @@ export default function WPCheckoutOrderSummary( {
 	);
 }
 
+function JetpackCheckoutSummaryPriceListPart( { responseCart }: { responseCart: ResponseCart } ) {
+	const translate = useTranslate();
+	const originalSubtotal = getOriginalSubtotal( responseCart );
+	const jetpackIntroDiscount = getJetpackIntroductoryDiscount( responseCart );
+	const jetpackBiYearlyDiscount = getJetpackBiYearlyDiscount( responseCart );
+	return (
+		<>
+			<CheckoutSummaryLineItem key="checkout-summary-line-item-plan-subscription">
+				<span>{ translate( 'Plan subscription' ) }</span>
+				<span>
+					{ formatCurrency( originalSubtotal, responseCart.currency, {
+						isSmallestUnit: true,
+						stripZeros: true,
```

Comment:
Why do we strip zeros here but not in either value below?

Reply:
That's right, we can keep it coherent and strip zeros here. I was biased by the design that was stripping zeros only for discounts.